client/src/pages/FavoriteDestinantions.js

```diff
@@ -1,11 +1,40 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import { Typography, Grid, Button } from "@material-ui/core";
-import places from "../database/places";
 import useStyles from "../styles/FavoriteDestinations";
 import FavoriteCheckBox from "../components/FavoriteCheckBox";
 
 function FavoriteDestinantions() {
   const classes = useStyles();
+  const [places, setPlaces] = useState([])
+
+
+
+  useEffect(() => {
+    async function getData() {
+      try {
+        let favoriteList = [];
+        //Assuming that the explore page can be accessed by both authenticated and unAuthenticated user.
+        const cityListResponse = await fetch("/api/cities", {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        });
+        const cityList = await cityListResponse.json();
+        cityList.map(city => {
+          city.favorite = favoriteList.indexOf(city.name) >= 0;
```

Comment:
`favoriteList` here is just an empty array, the value of `favoriteList.indexOf(city.name) >= 0` is always going to be `false`.

Reply:
Will effect the changes and corrections on next push.